tests/src/python/test_qgsserver_wms_getprint.py

```diff
@@ -412,6 +412,32 @@ def test_wms_getprint_two_maps(self):
         r, h = self._result(self._execute_request(qs))
         self._img_diff_error(r, h, "WMS_GetPrint_TwoMaps")
 
+    def test_wms_getprint_atlas(self):
+        qs = "?" + "&".join(["%s=%s" % i for i in list({
+             "MAP": urllib.parse.quote(self.projectPath),
+             "SERVICE": "WMS",
+             "VERSION": "1.3.0",
+             "REQUEST": "GetPrint",
+             "TEMPLATE": "layoutA4",
+             "FORMAT": "png",
+             "CRS": "EPSG:3857",
+             "ATLAS_PK": "3",
```

Comment:
Just out of curiosity, how do we indicate that we want all PK?

Reply:
> Do you mean if you want to request an atlas print with all the features in the layer?

Yes.

> This is not provided, you can however request several sheets by providing multiple values, e.g. ATLAS_PK=3,4,5,7,12

Do you think we could add some kind of notation to do that?

Reply:
It seems dangerous, someone could bring the server down with only a few requests. Maybe we should even make a restriction on the number of features that can be requested at a time, it could be a server project setting.

Reply:
> It seems dangerous, someone could bring the server down with only a few requests. 

Indeed, you're right.

> Maybe we should even make a restriction on the number of features that can be requested at a time, it could be a server project setting.

But if we have a restriction on the number of features, we could definitely have a notation to request all features without having to worry.

For example, if the restriction is `20` features and if the user wants 15 features, he could just do something like `ATLAS_PK=*` instead of `ATLAS_PK=0,1,2,3,4,.......`. What do you think?

Reply:
That makes sense and '*' seems to be a good notation. If the server restriction is lower than the number of features, the server will just send back the first x features

Reply:
> If the server restriction is lower than the number of features, the server will just send back the first x features

It sounds good, thanks @mhugent.